include/emp/Evolve/OEE.hpp

```diff
@@ -1,235 +1,245 @@
-#ifndef EMP_OEE_STATS_H
-#define EMP_OEE_STATS_H
+/**
```

Comment:
Did this file actually previously begin with an #endif??  That seems pretty bad...